bin/gemini

```diff
@@ -33,7 +33,9 @@ use Symfony\Component\Console\Application;
 
 
 try {
-    new Config(\dirname(__DIR__).'/env/config.xml', \dirname(__DIR__));
+    $dir = \dirname(__DIR__);
+    //new Config(\dirname(__DIR__).'/env/config.xml', \dirname(__DIR__));
+    Config::getInstance()->initialize($dir . '/env/config.xml', $dir);
```

Comment:
This will have to be revised now due to my changes to your branch we worked on in `kobens-core`.

```suggestion
    $config = Config::getInstance();
    $config->setRootDir(\dirname(__DIR__));
    $config->setConfig(\dirname(__DIR__).'/env/config.xml');
```

Reply:
I introduced $dir there because it was calling dirname twice which is typically bad practice.  Micro as it may be, it's just habit.  PS, why doesn't setConfig() just _use_ the $rootDir then you don't have to pass it in anymore.  Makes most sense to me!

Reply:
Hm. You mean maybe... this?

- Call `->setRootDir()` *once*
- By setting the root dir, we now know where the config is? No need for calling `->setConfig()` ever?

I think if I remember correctly. I wanted the ability to have the config be stored outside of the app if desired... maybe for some obscure security reason but at the end of the day it maybe doesn't make much sense to. Security through obscurity is not security. And I don't really know where else I'd put the `config.xml` file for that matter.